packages/compiler-cli/test/compliance/mock_compile/BUILD.bazel

```
@@ -0,0 +1,19 @@
+load("//tools:defaults.bzl", "ts_library")
+
+ts_library(
+    name = "mock_compile",
+    testonly = 1,
```

I'm seeing both `True` and `1` being used, but `True` makes more sense to me:

```suggestion
    testonly = True,
```

---

Indeed that seems correct https://docs.bazel.build/versions/master/be/common-definitions.html